jepsen/src/jepsen/nemesis/combined.clj

```diff
@@ -35,6 +45,8 @@
      :minority      - Chooses a random minority of nodes
      :majority      - Chooses a random majority of nodes
      :primaries     - All nodes which we think are primaries
+     :follower      - Chooses a single random node that we don't think is primary
+     :followers     - Chooses a random, non-empty subset of nodes we don't think are primaries
```

Comment:
For regularity with primaries, I suggest calling this some-secondaries.

Reply:
Fixed.